axe.d.ts

```diff
@@ -4,36 +4,54 @@
 
 declare module axe {
 
-	type ImpactValue = "minor" | "moderate" | "serious" | "critical";
+	enum ImpactValue {
+		minor = "minor",
+		moderate = "moderate",
+		serious = "serious",
+		critical = "critical"
+	}
 
-	type TagValue = "wcag2a" | "wcag2aa" | "section508" | "best-practice";
+	enum TagValue {
+		wcag2a = "wcag2a",
+		wcag2aa = "wcag2aa",
+		section508 = "section508",
+		"best-practice"
+	};
 
-	type ReporterVersion = "v1" | "v2";
+	enum ReporterVersion {
+		v1 = "v1",
+		v2 = "v2"
+	};
 
-	type RunOnlyType = "rule" | "rules" | "tag" | "tags";
+	enum RunOnlyType {
+		rule = "rule",
+		rules = "rules",
+		tag = "tag",
+		tags = "tags"
+	};
 
-	type resultGroups = "inapplicable" | "passes" | "incomplete" | "violations";
+	type ResultGroups = "inapplicable" | "passes" | "incomplete" | "violations";
 
 	type RunOnlyObject = {
```

Comment:
This is a confusing name if it isn't being used in runOnly (only in ElementContext)

Reply:
It'd be a breaking change to rename it tho, right?